src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Modal/BitModal.scss

```
@@ -33,16 +32,43 @@
     position: absolute;
     pointer-events: all;
     background-color: $clr-bg-overlay;
+    // The backdrop is put into the page in the same frame as the content it dims, so it fades in with it
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)